R/optionalInput.R

```diff
@@ -24,27 +19,17 @@
 #' @param fixed (`logical(1)` optional)\cr
 #'  whether to block user to select choices.
 #'
+#' @param fixed_on_single (`logical(1)` optional)\cr
+#'  whether to block user to select a choice when there is only one or less choice.
+#'  When `FALSE`, user is still able to select or deselect the choice.
```

Comment:
```suggestion
#'  Specifies whether to prevent the user from selecting choices when there is only one or less choice.
#'  When `FALSE`, the user is still able to select or deselect the choice.
```